.github/workflows/publish-api-dokkasaurus.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Dokkasaurus docs
+
+on:
+  push:
+    branches:
+      types: [released]
```

Comment:
We should run this action on every PR (Before we merge code into develop), and then add channes as a new commit to the PR

Reply:
This is a bit dangerous... It may reach a state that it enters a loop. The CI commits in the branch, then it starts again... Do some change that changes the code and restarts again... and again... and again... Although it should probably stop once there's no changes in the documentation, there's always the risk to enter a loop. Perhaps we should do it on every merge in `develop`?

Reply:
What about to have a check here that see if there is any needed update on the docs?
We can block the PR is the docs is not up-to-date, asking us (devs) to update it before merge the PR (Mainly the same we do with `./gradlew apiDump`).

The reason to do that is that we need to be sure code go to develop is ready to be released anytime. We can't merge code to develop that needs some changes before it is public released into a new version.

We should have some script (Maybe the one that you did on this PR) to update the docs locally from our machine before we push it to GH

Reply:
Now there's only a check that the documentation is up-to-date. The developers now can ran the ./script/dokkasaurus.sh